src/syscalls/timer.c

```diff
@@ -56,7 +56,7 @@ u32 svcSetTimer()
     u32 timer = arm11_R(0);
     u64 initial = arm11_R(1) | ((u64)arm11_R(2) << 32);
     u64 interval = arm11_R(3) | ((u64)arm11_R(4) << 32);
-    DEBUG("SetTimer --todo-- %08x %08x %08x\n", timer, initial, interval);
+    DEBUG("SetTimer --todo-- %08x %08llx %08llx\n", timer, initial, interval);
```

Comment:
src/syscalls/timer.c:59:40: error: expected ')'
    DEBUG("SetTimer --todo-- %08x %08" PRIx64 " %08" PRIx64 "\n", timer, initial, interval);
                                       ^
src/syscalls/timer.c:59:5: note: to match this '('
    DEBUG("SetTimer --todo-- %08x %08" PRIx64 " %08" PRIx64 "\n", timer, initial, interval);
    ^


I wrote :
DEBUG("SetTimer --todo-- %08x %08" PRIx64 " %08" PRIx64 "\n", timer, initial, interval);